src/js/Signature.jsx

```diff
@@ -3,7 +3,7 @@ var api = require('./api');
 
 var Signature = React.createClass({
   propTypes: {
-    setAppStatus: React.PropTypes.func.isRequired,
+    appStatus: React.PropTypes.object.isRequired,
```

Comment:
See comment earlier about using `objectOf` but also, is there a reason that this has `isRequired` and the others don't?